docs/docs/configuration.md

```diff
@@ -311,6 +311,46 @@ The file name of the library. The main bundle of the library will be output to `
 
 If `libraryFile` is not specified then `libraryName` will be used instead.
 
+## pathAliases
+
+Type: `Record<string, string>`
+
+Default: `{ 'src/*': './src/*' }`
+
+**Only available with Vite bundler**
+
+Custom path alias mappings for module resolution. Each alias pattern maps to a destination path relative to the project root.
+
+When using the `vite` bundler, sku provides a default `src/*` alias that maps to `./src/*`. The `pathAliases` option allows you to define additional custom aliases.
```

Comment:
We've already mentioned that this is exclusive to Vite so I think this can be removed.

```suggestion
sku provides a default `src/*` alias that maps to `./src/*`. The `pathAliases` option allows you to define additional custom aliases.
```